### ISSUE TITLE
Instant Death Mid-Day

### ISSUE THREAD

**Issue by aikonamika** [HabitRPG/habitica#3880]
I had completed several To-Dos earlier in the day, having logged in and taken damage from the boss and my unchecked Dailies like normal. Approximately ten minutes ago I checked off another To-Do, and promptly died. I checked my Party page and found that no damage had occurred to me from the boss since 7 hours ago, and my Custom roll-over for the day is currently set at 03:00. I can't find any cause for the sudden death as of right now!

**Comment by lefnire** [comment]
Could you try a refresh on the party page? Chat notifs from boss battles don't show up automatically. Had your dailies which you checked today become unchecked when that happened?

**Comment by aikonamika** [comment]
The dailies became unchecked when I first refreshed for the day. I checked the Party page and the most recent damage was 12 hours ago, where I dealt Vice's Shade 0.0 damage (odd, as I know that I'd done _some_ dailies yesterday) and he dealt the party 12.9 damage. Which shouldn't have killed me, as I had 19 health left _after_ that message showed up for the party chat.

I refreshed both my browser and by clicking the "Fetch Recent Messages" buttons.
